WalletWasabi.Fluent/ViewModels/Dialogs/ShowQrCameraDialogViewModel.cs

```diff
@@ -15,6 +15,7 @@ namespace WalletWasabi.Fluent.ViewModels.Dialogs
 	public partial class ShowQrCameraDialogViewModel : DialogViewModelBase<string?>
 	{
 		[AutoNotify] private WriteableBitmap? _qrImage;
+		[AutoNotify] private string _message = "";
```

Comment:
This variable is not used.

Reply:
It is. `[AutoNotify]` creates a public property in the background, which is changed in line 52 at
```
if (string.IsNullOrWhiteSpace(Message))
{
	Message = $"Invalid content. Please make sure the QR code contains a correct {network.Name} address.";
}
```